.github/workflows/performance-benchmarks-naslib.yml

```diff
@@ -0,0 +1,197 @@
+name: Performance Benchmarks with naslib
+
+on:
+  workflow_dispatch:
+    inputs:
+      sampler-list:
+        description: 'Sampler List: A list of samplers to check the performance. Should be a whitespace-separated list of Optuna samplers. Each sampler must exist under `optuna.samplers` or `optuna.integration`.'
+        required: false
+        default: 'RandomSampler TPESampler'
+      sampler-kwargs-list:
+        description: 'Sampler Arguments List: A list of sampler keyword arguments. Should be a whitespace-separated list of json format dictionaries.'
+        required: false
+        default: '{} {\"multivariate\":true,\"constant_liar\":true}'
+      pruner-list:
+        description: 'Pruner List: A list of pruners to check the performance. Should be a whitespace-separated list of Optuna pruners. Each pruner must exist under `optuna.pruners`.'
+        required: false
+        default: 'NopPruner'
+      pruner-kwargs-list:
+        description: 'Pruner Arguments List: A list of pruner keyword arguments. Should be a whitespace-separated list of json format dictionaries.'
+        required: false
+        default: '{}'
+      n-runs:
+        description: 'Number of Studies'
+        required: false
+        default: '10'
+
+
+jobs:
+  performance-benchmarks-naslib:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Setup Python3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+
+    - name: Install gnuplot
+      run: |
+        sudo apt update
+        sudo apt -y install gnuplot
+
+    - name: Setup cache
+      uses: actions/cache@v2
+      env:
+        # Caches them under a common name so that they can be used by other performance benchmark.
+        cache-name: performance-benchmarks
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-3.9-${{ env.cache-name }}-${{ hashFiles('**/setup.py') }}-v1
+        restore-keys: |
+          ${{ runner.os }}-3.9-${{ env.cache-name }}-${{ hashFiles('**/setup.py') }}
+
+    - name: Install Python libralies
+      run: |
+        python -m pip install --upgrade pip
+        pip install --progress-bar off -U setuptools
+
+        # Install minimal dependencies and confirm that `import optuna` is successful.
+        pip install --progress-bar off .
+        python -c 'import optuna'
+        optuna --version
+
+        pip install --progress-bar off .[benchmark]
```

Comment:
This line can be removed since this *benchmark* is corresponding to *speed* benchmark, not kurobako benchmark. 
```suggestion
```